modules/vpc-peering/pages/azure.adoc

```diff
@@ -0,0 +1,48 @@
+= Set up VN Peering on Azure
+:description: The process to set up VPC peering between the TigerGraph Cloud Azure VN and another Azure VN.
+:experimental:
+
+
+This is a guide for setting up a Virtual Network(VN) peering connection between the TigerGraph Cloud VN and another Azure VN.
+
+
+== Prerequisite
+* At least one unterminated TigerGraph Cloud solution on GCP
+
+== Procedure
+
+=== 1. Choose the Azure VN to initiate peering from
+
+From the Cloud Portal, navigate to menu:Network[VPC Peering].
+From the platform list, click btn:[Initiate a VPC Peering].
+
+image::cloud-network.png[Network Tab]
+
+In the pop-up window, choose Azure as your network platform.
+Then choose the region of the platform you want to initiate the peering connection from.
+Click btn:[Next].
+
+=== 2. Add TigerGraph Cloud to your Azure tenant
+In the Peered VPC tab, fill in your link:https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenan[Azure tenant ID] and your VN ID (Resource ID for your VN).
+Click btn:[Next].
+
+Click on the link in the instructions to add TigerGraph Cloud to your tenant.
+Doing so opens a new tab, and if authorization is successful, it will redirect you back to the cloud portal.
+You can now safely close this tab.
+
+=== 3. Assign Network Contributor role to TigerGraph Cloud VN Peerer
+From the Azure portal, navigate to btn:[Virtual Networks] and choose the network you are peering with.
+Then navigate to menu:Access Control[Add > Add role assignment].
+
+image::role-assignment.png[Role assignment]
+
+From the list of roles, choose Network Contributor.
+Click btn:[Next].
+
+Click btn:[Select Members] and search for TigerGraph VN Peerer.
+Select and click btn:[Review + assign].
+You'll see a notification if the role assignment is successful.
+
+=== 4. Initiate peering
+Go back to the Cloud Portal, and click btn:[Initiate Peering].
+If connection is successful, you'll see the connection listed under Azure and its status is "Connected".
```

Comment:
If the connection is successful, you'll see the connection listed under Azure with its status shown as "Connected".